presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -51,6 +51,7 @@
 import static com.google.common.base.Preconditions.checkState;
 import static com.google.common.collect.ImmutableList.toImmutableList;
 import static io.airlift.units.DataSize.Unit.BYTE;
+import static java.lang.Math.min;
```

Comment:
Shouldn't getBlockedQueuedQueries be added to ResourceGroup and then overridden in InternalResourceGroup? Am I misunderstanding?